api/v2/v2.go

```diff
@@ -87,6 +87,12 @@ var (
 	// timestamp, block height & tx id
 	LastAnchorRoute = RoutePrefix + "/last"
 
+	// LastDigestsRoute defines the API route for retriving
+	// the last last n digests the client wants. If n is bigger
```

Comment:
There needs to be limits to the number of digests a client can request, otherwise a client could request the entire dcrtime history and DoS the server.